File: p1/Fibonacci.cpp
#include <iostream>
using namespace std;

void fibonacci(){
	double a[100];
	int i,;
	a[0]=1;
	a[1]=1;
	cout<<"Fibonacci数列前100项数如下"<<endl; 
	cout<<"第1项为 1"<<endl;
    cout<<"第2项为 1"<<endl;
	for(i=2;i<100;i++) 
	{ 
		a[i]=a[i-1]+a[i-2];
		cout<<"第"<<i+1<<"项是 " ;
		cout<<a[i]<<endl;
	}
}
int main(){
	fibonacci();
	return 0;
}

File: p1/Fibonacci.java
package first;
import java.math.BigInteger; 

public class Fibonacci { 
	
		 public static void main(String[] args) {
			 BigInteger a[] = new BigInteger[100];
			  a[0]=new BigInteger("1");
			  a[1]=new BigInteger("1");
			  for (int i = 2; i < a.length; i++) {
			   a[i] = a[i - 1].add(a[i - 2]) ; }
			  int j;
			  System.out.println("斐波那契数列的前100项如下所示：");
			  for (int i = 0; i < a.length; i++) {
				  j=i+1;
				  System.out.print("第"+ j +"项是");
				  System.out.println(a[i]);
			  }
		 }
		 
}
